support-frontend/assets/components/headers/header/header.jsx

```diff
@@ -20,6 +21,7 @@ export type PropTypes = {|
   utility: Option<Node>,
   countryGroupId: ?CountryGroupId,
   displayNavigation: boolean,
+  displayCheckout?: boolean | void,
```

Comment:
```suggestion
  display: 'navigation' | 'checkout' | null
```

i'd do something like this instead of two boolean parameters, since they are mutually exclusive (you can't display navigation AND display checkout)

Reply:
Thanks @walaura - shall do